compact_forms.module

```diff
@@ -116,10 +116,11 @@ function compact_forms_form_alter(&$form, $form_state, $form_id) {
  */
 function _compact_forms_process_fields(&$form, $config) {
   $description = $config->get('description');
-  $required = $config->get('required');
+  $required_marker = $config->get('required');
   $size = $config->get('size');
 
   foreach (element_children($form) as $key) {
+    (!empty($form[$key]['#required'])) ? $required = $form[$key]['#required'] : $required = 0;
```

Comment:
@laryn This line should be as follows:

`$required = isset($form[$key]['#required']) ? $form[$key]['#required'] : FALSE;`

And put a blank line underneath it too for readability please.